documentation/development-docs/vm-for-partners.md

```diff
@@ -0,0 +1,126 @@
+        try {
+            $sa = Get-AzureRmStorageAccount -StorageAccountName $StorageAccountName -ResourceGroupName $ResourceGroupName
+        } catch {
+            if ($_ -like "*was not found*") {
+                $sa = New-AzureRmStorageAccount -ResourceGroupName $ResourceGroupName -Name $StorageAccountName -Location $Location -SkuName Standard_LRS 
+                Write-Host "`tCreated storage account $StorageAccountName." -ForegroundColor Yellow
+            } else {
+                throw $_.Exception
+            }
+        }
+
+        $script:StorageAccountContext = $sa.Context
+
+        # Storage Container
+        Write-Host "`tStorage Container"
+        try {
+            $sc = Get-AzureStorageContainer -Name $StorageContainerName -Context $StorageAccountContext
+        } catch {
+            if ($_ -like "*Can not find the container*") {
+                $sc = New-AzureStorageContainer -Name $StorageContainerName -Context $StorageAccountContext  
+                Write-Host "`tCreated storage container $StorageContainerName." -ForegroundColor Yellow
+            } else {
+                throw $_.Exception
+            }
+        }
+    }
+
+    function CopyVhdBlob() {
+        Write-Host "==> Start-AzureStorageBlobCopy"
+        $null = Start-AzureStorageBlobCopy -AbsoluteUri $VhdSasUri -DestContainer $StorageContainerName -DestContext $StorageAccountContext -DestBlob $VhdFileName
+        $Time = [System.Diagnostics.Stopwatch]::StartNew()
+        Write-Host "==> Polling Azure Storage Blob Copy State..."
+        $null = Get-AzureStorageBlobCopyState -Blob $VhdFileName -Container $StorageContainerName -Context $StorageAccountContext -WaitForComplete    
+        $CurrentTime = $Time.Elapsed
+        $Elapsed = $([string]::Format("{0:d2}:{1:d2}:{2:d2}", $CurrentTime.hours, $CurrentTime.minutes, $CurrentTime.seconds))
+        Write-Host "`tVHD blob has been copied. Time: $Elapsed" -ForegroundColor Yellow
+    }
+
+    function CreateImageFromVhd() {
+        Write-Host "==> CreateImageFromVhd"
+        $Blob = Get-AzureStorageBlob -Context $StorageAccountContext -Container $StorageContainerName -Blob $VhdFileName
+        $BlobUri = $Blob.ICloudBlob.Uri.AbsoluteUri
+
+        $ImageConfig = New-AzureRmImageConfig -Location $Location
+        $ImageConfig = Set-AzureRmImageOsDisk -Image $ImageConfig -OsType Windows -OsState Generalized -BlobUri $BlobUri
+        $script:Image = New-AzureRmImage -ImageName $ImageName -ResourceGroupName $ResourceGroupName -Image $ImageConfig
+    }
+
+    function CreateVmFromImage() {
+        Write-Host "==> CreateVmFromImage"
+        New-AzureRmVm -Name $VmName -Location $Location -ResourceGroupName $ResourceGroupName -ImageName $Image.Id -Credential $VmCredential -Size $VmSize
+    }
+
+    SelectSubscription
+    CreateResourceGroupIfNone
+    CreateStorageIfNone
+    CopyVhdBlob
+    CreateImageFromVhd
+    CreateVmFromImage
+
+    Write-Host "==> All done."
+    ```
```

Comment:
You might mention how to start a VM (on the portal probably is easiest).  Also, can you add the instructions that I created for setting up the machine for your personal github account: https://github.com/maddieclayton/azure-powershell/wiki/new-doc

Reply:
> You might mention how to start a VM (on the portal probably is easiest). 

I think there are a lot of documents on the topic on the Internet.
> Also, can you add the instructions ...

Do you want me to add the instruction as a link or copy-paste? The link you provided refers to a doc in your personal fork.

Reply:
Just copy paste